view/theme/frio/templates/settings_features.tpl

```diff
@@ -19,7 +19,11 @@
 				<div id="{{$g}}-settings-content" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{$g}}-settings-title">
 					<div class="section-content-tools-wrapper">
 						{{foreach $f.1 as $fcat}}
+							{{* Here is an addional condition to filter the richtext feature
+							because frio doesn't support the richtext *}}
+							{{if $fcat.0 != 'feature_richtext'}}
```

Comment:
I don't think we should hide the setting from users. Instead, we should just make it clear that it won't work with all the themes.